lms/views/dashboard/views.py

```diff
@@ -1,10 +1,14 @@
+from datetime import timedelta
+
 from pyramid.httpexceptions import HTTPFound
 from pyramid.view import forbidden_view_config, view_config
 
 from lms.security import Permissions
 from lms.services import OrganizationService
 from lms.validation.authentication import BearerTokenSchema
 
+AUTHORIZATION_DURATION_SECONDS = 60 * 60 * 24 * 7  # One week
```

Comment:
I was thinking that, perhaps we could define this constant in the `js_config` module, so that it can be used there directly, instead of passing it from here in the three different places where we call `js_config.enable_dashboard_mode(...)`.

Then, we import that constant here to use it when calling `BearerTokenSchema(...).authorization_param(...)`.

Just an idea, it's not terrible as it is.

Reply:
In this case I'm leaning towards the view owning this value, it does have to call enable_dahboard multiple times with the exact same values but I reckon that's an issue on it's own.

We could further type the jsconfig response, to include this value in the backend so the types start propagating outside the dashboard mode, I'll do that in a follow up PR.

Reply:
> it does have to call enable_dahboard multiple times with the exact same values but I reckon that's an issue on it's own

Yep, that's a good point.